tests/unit/AgmExtension.spec.ts

```diff
@@ -196,14 +196,21 @@ describe('AGM Extension view', () => {
       agmExtension: {
         agmDueDate: null,
         agmYear: null,
+        alreadyExtended: null,
         currentDate: '2023-11-06',
+        expireDateApprovedExt: null,
+        expireDateCurrExt: undefined,
+        extReqForAgmYear: undefined,
         extensionDuration: NaN,
         incorporationDate: new Date('2000-01-01T08:00:00.000Z'),
+        intendedAgmDate: undefined,
         isEligible: true,
         isFirstAgm: null,
         isGoodStanding: true,
-        alreadyExtended: null,
-        requestExpired: null
+        prevAgmRefDate: undefined,
+        requestExpired: null,
+        totalApprovedExt: NaN,
+        year: null
```

Comment:
These are all the properties in the JSON filing data object (ie, what gets posted to the Legal API).